### PR TITLE
Adding select2 to country and business sector dropdowns

### PR DESCRIPTION
**Pull request by mauricioac**
- Adding select2 to the dropdows mentioned on the title
- Tested without JS and forms work
- Tested required validation and it works with select2 as well

### CODE REVIEW

File: app/assets/javascripts/frontend/select2_dropdowns.js.coffee [hunk 1]
```diff
@@ -0,0 +1,5 @@
+window.Select2Dropdowns =
```
Comment (matthewford) [inline app/assets/javascripts/frontend/select2_dropdowns.js.coffee]: Line ends with trailing whitespace

File: app/assets/stylesheets/frontend/forms.scss [hunk 2]
```diff
@@ -1045,6 +1045,25 @@ input[type="file"] {
   }
 }
 
+// Select2 dropdowns
+.select2-container {
+  min-width: 14em;
+  
+  .select2-selection__rendered {
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: Selector `select2-selection__rendered` should be written in lowercase with hyphens

File: app/assets/stylesheets/frontend/forms.scss [hunk 3]
```diff
@@ -1045,6 +1045,25 @@ input[type="file"] {
...
+    font-size: 14px;
+  }
+
+  .select2-search__field {
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: Selector `select2-search__field` should be written in lowercase with hyphens

File: app/assets/javascripts/frontend/select2_dropdowns.js.coffee [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+  init: ->
+    console.log "really?"
```
Comment (dreamfall) [inline app/assets/javascripts/frontend/select2_dropdowns.js.coffee]: Please, remove `console.log`

**Pull request merged**